aten/src/ATen/native/cpu/SumKernel.cpp

```diff
@@ -1,11 +1,12 @@
-#include <ATen/Dispatch.h>
-#include <ATen/native/TensorIterator.h>
 #include <ATen/native/ReduceOps.h>
-#include <ATen/native/cpu/Reduce.h>
-#include <c10/util/llvmMathExtras.h>
 
 #include <algorithm>
 
+#include <ATen/Dispatch.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Reduce.h>
+#include <ATen/native/cpu/utils.h>
+#include <c10/util/llvmMathExtras.h>
```

Comment:
nit: looks like llvmMathExtras are not needed here, only by cpu/utils.h?

Reply:
Done.